entrypoint.sh

```diff
@@ -17,13 +17,32 @@ _RUNNER_NAME=${RUNNER_NAME:-${RUNNER_NAME_PREFIX:-github-runner}-$(head /dev/ura
 _RUNNER_WORKDIR=${RUNNER_WORKDIR:-/_work}
 _LABELS=${LABELS:-default}
 _RUNNER_GROUP=${RUNNER_GROUP:-Default}
-_SHORT_URL=${REPO_URL}
 _GITHUB_HOST=${GITHUB_HOST:="github.com"}
 
-if [[ ${ORG_RUNNER} == "true" ]]; then
```

Comment:
For now we should find a way to make it work as-is (with ORG_RUNNER) and as new (RUNNER_SCOPE )

There are quite a few users of this and I'd like to throw some deprecation warnings + a timeline before making a hard change.